core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -1634,6 +1634,17 @@ default Optional<TopNApplicationResult<ConnectorTableHandle>> applyTopN(
         return Optional.empty();
     }
 
+    /**
+     * If the file is ordered, attempt to push down sort or topN to connector.
```

Comment:
On the SPI level, we avoid talking about _files_, since underlying connector can store data in any way.

Reply:
done.